assets/src/components/button/style.scss

```diff
@@ -0,0 +1,53 @@
+/**
+ * Button styles,
+ */
+
+.muriel-button {
+	text-align: center;
+	font-size: 14px;
+	line-height: 18px;
+	font-weight: 500;
+	color: $muriel-gray-700;
+	text-shadow: none;
+
+	&.is-button {
+		display: inline-block;
+		background-color: $muriel-white;
+		border-color: #CCCCCC;
+		border-radius: 3px;
+		cursor: pointer;
+		height: auto;
+		padding: 13px;
+		margin: 14px 7px;
+
+		&:active:enabled {
+			border-width: 2px 1px 1px;
+		}
+	}
+
+	&.is-primary {
+		border: none;
+		background-color: $muriel-hot-pink-500;
+		box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25);
+		color: $muriel-white;
+		text-shadow: none;
+
+		&:hover,
+		&:focus:enabled {
+			background-color: $muriel-hot-pink-400;
```

Comment:
To my eye, the black line under the Primary button in hover state looks funny, and since we don't have a model for hover states (that I'm aware of?) we can make styling decisions here. My comment is only about Primary—I like the hover state of the Default button with the same shadow... Adding `box-shadow: inherit;` to the focus block would resolve it.  

<img width="469" alt="Screen Shot 2019-05-14 at 8 16 58 AM" src="https://user-images.githubusercontent.com/1477002/57697452-3e232600-7621-11e9-91c4-4df850241815.png">